README.md

```diff
@@ -116,6 +117,26 @@ type DatabaseService = {
 };
 ```
 
+```ts
+type MongoService = {
+  type: 'MONGO';
+  serviceName: string;
+  environment?:
+    | aws.ecs.KeyValuePair[]
+    | ((services: Services) => aws.ecs.KeyValuePair[]);
+  secrets?: aws.ecs.Secret[] | ((services: Services) => aws.ecs.Secret[]);
+  port: pulumi.Input<number>;
```

Comment:
Should we have a port here optional and use the default mongo port if it is not provided?